Makefile

```diff
@@ -140,6 +145,12 @@ NGC_PYTORCH_HPC_REPO := pytorch-ngc-hpc-dev
 NGC_TF_REPO := tensorflow-ngc-dev
 NGC_TF_HPC_REPO := tensorflow-ngc-hpc-dev
 
+INFINITYHUB_PYTORCH_PREFIX := rocm/pytorch
+INFINITYHUB_TENSORFLOW_PREFIX := rocm/tensorflow
+INFINITYHUB_PYTORCH_VERSION := 2.1.2
```

Comment:
It looks like these are not used.

Reply:
Oh, is `INFINITYHUB_PYTORCH_PREFIX` intended to be used at line 184: BASE_IMAGE="rocm/pytorch:rocm6.1_ubuntu22.04_py3.10_pytorch_2.1.2" \
                 --build-arg

Reply:
Removed these